### PR TITLE
Escape Repository as a Service Proposal

### PR DESCRIPTION
**Pull request by garciagenrique**

### CODE REVIEW

File: _gsocprojects/2022/project_ESCAPE-OSSR.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  It will enable a true multi-messenger data-driven cooperative approach based 
+  on the FAIR principle requirements and will become part of the EOSC global 
+  catalogue of services. In a collaborative effort of all ESCAPE partners, common 
+  and innovative approaches will be fostered. 
+  The software, science tools & other services available on ESCAPE OSSR are 
+  openly available to all and allow users to access multi-messenger and 
+  multi-domain software to mine open-access datasets. This guarantees 
+  cross-fertilisation and an open innovation environment for science 
+  interoperability and software re-usage.
+---
```
Comment (agheata) [inline _gsocprojects/2022/project_ESCAPE-OSSR.md:22]:
```suggestion
---
summary: |
  Add a shorter description of one paragraph only

```

File: _gsocproposals/2022/proposal_ESCAPE-repository-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+* Create a JupyterLab extension that connects the Jupyter environment and Zenodo (Repository) using the  
+eOSSR python library to display the repository records.
+* Participate in the development of the eOSSR library to integrate the developed extension. 
+* Bring back the record content to the Jupyter environment using the developed extension.
+* Investigate the possibility of installing the different software records available in the repository 
+and find solutions to the different software languages libraries.
+* Define the needed stages to publish back into Zenodo and the ESCAPE community.
+
+
```
Comment (agheata) [inline _gsocproposals/2022/proposal_ESCAPE-repository-as-a-service.md:37]: It's recommended to add at least one sentence about what the student will learn.

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_ESCAPE-repository-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Repository as a Service
+layout: gsoc_proposal
+project: ESCAPE-OSSR
+year: 2022
+difficulty: medium
+duration: 300
```
[the remark this answers is not in the document]
Comment (garciagenrique) [reply _gsocproposals/2022/proposal_ESCAPE-repository-as-a-service.md]: Right, thanks !

**Comment by garciagenrique** [comment]
Thanks for the feedback @agheata !
Comments are implemented.

Setting temporarily the PR on draft (contacting HSF admins by email)

**Comment by garciagenrique** [comment]
(I edited the previous message).

PR is ready for re-review. Thanks in advance :)

**Review by agheata — APPROVED** [review]

**Comment by garciagenrique** [comment]
Hi @agheata,
I see that the Organization, project and proposal were not correctly deployed in the HSF page. Let me know if I can help anyhow.

**Comment by aflp91** [comment]
@garciagenrique: I opened a PR #1103 about this yesterday…

**Comment by garciagenrique** [comment]
> @garciagenrique: I opened a PR #1103 about this yesterday…

Thanks a lot @aflp91, didn't got any notification.

**Pull request merged**